src/api/serializer.py

```diff
@@ -205,28 +238,28 @@ def describe_table(self, repo, table, detail=False):
         res = self.manager.describe_table(
             repo=repo, table=table, detail=False)
 
-        response = []
+        columns = []
         for column in res:
             response_obj = {}
             response_obj['column_name'] = column[0]
             response_obj['data_type'] = column[1]
-            response.append(response_obj)
-        return {'columns': response}
+            columns.append(response_obj)
+
+        res = self.manager.list_table_permissions(repo, table)
+        permissions = [permission for sublist in res for permission in sublist]
+
+        return {'columns': columns, 'permissions': permissions}
 
     def delete_table(self, repo, table, force=False):
         success = self.manager.delete_table(repo, table, force)
         return success
 
     def export_table(self, repo, table, file_format='CSV', delimiter=',',
                      header=True):
-
-        success = DataHubManager.export_table(
-            username=self.username, repo_base=self.repo_base,
+        self.manager.export_table(
```

Comment:
That could be a pretty large response, since we don't paginate file responses. I'm inclined to leave it as is and open an issue that the export_* responses should include a link to the file they created.